static/app/views/issueDetails/traceTimeline/useTraceTimelineEvents.tsx

```diff
@@ -17,11 +17,11 @@ interface BaseEvent {
   transaction: string;
 }
 
-interface TimelineDiscoverEvent extends BaseEvent {}
```

Comment:
This interface should have had `event.type` and `stack.function` from the get-go.